src/main/res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">AnySoftKeyboard - change_me Language Pack</string>
-    <string name="keyboard_name">change_me keyboard</string>
-    <string name="dictionary_name">change_me</string>
+    <string name="app_name">AnySoftKeyboard - Indonesian Language Pack</string>
+    <string name="keyboard_name">Indonesian keyboard</string>
```

Comment:
You mean from `Indonesian keyboard` to `Indonesian` only, in line 4 of string.xml ?

I am sorry for the mistake, I learn some set up from [pali](https://github.com/yuttadhammo/ask-pali-keyboard/blob/master/res/values/strings.xml) and other language branch, and they use `foo keyboard`.

I will change it immidiately after your answer, I am asking about it because I am worry I missunderstand your order.